src/components/subscriptionDetail/TabView.tsx

```diff
@@ -0,0 +1,16 @@
+export const SUBSCRIPTION_VIEWTYPE_SELECTOR = "subscription-viewtype";
```

Comment:
Rename the file to include `types` or maybe move the contents to `utils/types`?

Reply:
Ended up moving the contents to `utils/types`